multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -446,20 +453,29 @@ write_files:
                       }
                     ],
                     "resources": {
+                      "requests": {
+                        "cpu": "10m", 
+                        "memory": "20Mi"
+                      }, 
                       "limits": {
                         "cpu": "10m",
                         "memory": "20Mi"
                       }
-                    }
+                    },
+                    "terminationMessagePath": "/dev/termination-log",
+                    "imagePullPolicy": "IfNotPresent"
                   }
                 ],
+                "restartPolicy": "Always",
+                "terminationGracePeriodSeconds": 30,
                 "dnsPolicy": "Default",
                 "volumes": [
                   {
                     "emptyDir": {},
                     "name": "etcd-storage"
                   }
-                ]
+                ],
+                "securityContext": {}
```

Comment:
I generated the JSON by round tripping the YAML through an actual running cluster, and it gave me back a bunch of default fields (including imagePullPolicy/securityDomain/etc)